wallet/src/de/schildbach/wallet/ui/preference/SettingsFragment.java

```diff
@@ -1,153 +0,0 @@
-    private Preference trustedPeerOnlyPreference;
-
-    private static final Logger log = LoggerFactory.getLogger(SettingsFragment.class);
-
-    @Override
-    public void onAttach(final Activity activity) {
-        super.onAttach(activity);
-
-        this.activity = activity;
-        this.application = (WalletApplication) activity.getApplication();
-        this.config = application.getConfiguration();
-        this.pm = activity.getPackageManager();
-    }
-
-    @Override
-    public void onCreate(final Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-
-        addPreferencesFromResource(R.xml.preference_settings);
-
-        backgroundThread = new HandlerThread("backgroundThread", Process.THREAD_PRIORITY_BACKGROUND);
-        backgroundThread.start();
-        backgroundHandler = new Handler(backgroundThread.getLooper());
-
-        btcPrecisionPreference = findPreference(Configuration.PREFS_KEY_BTC_PRECISION);
-        btcPrecisionPreference.setOnPreferenceChangeListener(this);
-
-        trustedPeerPreference = findPreference(Configuration.PREFS_KEY_TRUSTED_PEER);
-        ((EditTextPreference) trustedPeerPreference).getEditText().setSingleLine();
-        trustedPeerPreference.setOnPreferenceChangeListener(this);
-
-        trustedPeerOnlyPreference = findPreference(Configuration.PREFS_KEY_TRUSTED_PEER_ONLY);
-        trustedPeerOnlyPreference.setOnPreferenceChangeListener(this);
-
-        final Preference dataUsagePreference = findPreference(Configuration.PREFS_KEY_DATA_USAGE);
-        dataUsagePreference.setEnabled(pm.resolveActivity(dataUsagePreference.getIntent(), 0) != null);
-
-        updateTrustedPeer();
-    }
-
-    @Override
-    public void onDestroy() {
-        trustedPeerOnlyPreference.setOnPreferenceChangeListener(null);
-        trustedPeerPreference.setOnPreferenceChangeListener(null);
-        btcPrecisionPreference.setOnPreferenceChangeListener(null);
-
-        backgroundThread.getLooper().quit();
-
-        super.onDestroy();
-    }
-
-    @Override
-    public boolean onPreferenceChange(final Preference preference, final Object newValue) {
-        // delay action because preference isn't persisted until after this method returns
-        handler.post(new Runnable() {
-            @Override
-            public void run() {
-                if (preference.equals(btcPrecisionPreference)) {
-                    WalletBalanceWidgetProvider.updateWidgets(activity, application.getWallet());
-                } else if (preference.equals(trustedPeerPreference)) {
-                    application.stopBlockchainService();
-                    updateTrustedPeer();
-                } else if (preference.equals(trustedPeerOnlyPreference)) {
-                    application.stopBlockchainService();
-                }
-            }
-        });
-
-        return true;
-    }
-
-    private void updateTrustedPeer() {
-        final String trustedPeer = config.getTrustedPeerHost();
```

Comment:
This was once a useful function for debugging mostly, but with v7.0 the UI for it was removed.  I will make a note of this code in the event the Trusted Peer feature is returned.

In all likelihood all of the code in the `preference` package folder is obsolete.  Recently we restored the XPUB feature, but the other settings and diagnostic were either removed or moved to the new More menu structure.